webapp/src/js/controllers/inbox.js

```diff
@@ -151,6 +147,20 @@ var _ = require('underscore'),
 
       updateReplicationStatus(status);
     });
+    
+    const setAppTitle = () => {
+      ResourceIcons.getAppTitle().then(title => {
+        document.title = title;
+        $('.header-logo').attr('title', `${title} | ${APP_CONFIG.version}`);
+      });
+    };
+    setAppTitle();
+
+    Changes({
+      key: 'resource-icons',
```

Comment:
There's already a changes listener with this key so this one will overwrite the other one. Give it a unique key.